exs/mundo_1/c#/030.cs

```diff
@@ -0,0 +1,42 @@
+/*
+Desafio 030
+
+Problema:  Crie um programa que leia um número inteiro e mostre na tela se ele é PAR ou ÍMPAR.
+
+Resolução do problema:
+*/
+
+using System;
+
+class io {
+    public static string input(string text) {
+        Console.Write(text);
+        return Console.ReadLine();
+    }
+    
+    public static void print(object data, string end="\n") {
+        string text = data.ToString();
+
+        Console.Write($"{text}{end}");
+    }
+}
+
+class Program {
+  public static void Main () {
+    int num = int.Parse(io.input(
+      "Insira um número: "
+    ));
+
+    if (num < 0) {
+      io.print("Número inválido");
+    }
+
+    else if (num % 2 == 0) {
```

Comment:
Remova esse trecho de código.

Tente solucionar somente o que está descrito no enunciado. Não é solicitado tratativa de número inválido.

*Obs: A lógica dessa tratativa está errada, visto que números inteiros negativos também podem ser classificados como números ímpares e pares.*

```suggestion
    if (num % 2 == 0) {
```